docs/software/devtools/vihps.md

```diff
@@ -0,0 +1,138 @@
+[](){#ref-devtools-vihps}
+# VI-HPS tools
+
+The [VI-HPS](https://www.vi-hps.org/tools) Institute (Virtual Institute for High Productivity Supercomputing) provides tools that can assist developers of simulation codes to address their needs in performance analysis.
+
+## [Score-P](https://www.vi-hps.org/projects/score-p/overview/overview.html)
+
+![logo_scorep](../../images/devtools/vihps/logo_scorep.png){ width="5%"}
```

Comment:
Is the logo necessary? It is easier to read (and search) if the name of the package is in plain text instead of an inline image.

Images also increase the size of the repository, and we suggest only including them where they significantly improve the docs.

Reply:
we are talking about less than 250K here

Reply:
It all adds up over time.

But the main point is that using a picture where the name could be written in plain text makes the docs less readable and searchable.